supporting_scripts/course-scripts/quick-course-setup/README.md

```diff
@@ -166,6 +166,24 @@ The script will automatically perform all the necessary steps:
 5. Create a programming exercise or use an existing one. 
 6. Add participation and commit for each student.
 
+### Optional: Generating Different Results For All Created Students (Should only be done Locally!!)
+
+If you want to generate different results for all the students created by the script:
+
+1.	Navigate to the [testFiles](../../../src/main/resources/templates/java/test/testFiles) folder and copy the [RandomizedTestCases](./testFiles-template/randomized/RandomizedTestCases.java) file into it.
+    If you want, you can delete the existing folders (behavior and structural) from the programming exercise’s test case template. The new test cases will randomly pass or fail, causing different results for each student.
+2.	Rebuild Artemis to apply the changes.
+3.	Run the main method in large_course_main.py. Now, all created students should have varying results in the programming exercise.
+```shell
+python3 large_course_main.py
+```
+4.  Make sure to revert these changes after running the script. You can copy the original test case files from the [default](./testFiles-template/default) folder back into the [testFiles](../../../src/main/resources/templates/java/test/testFiles) folder and delete the [RandomizedTestCases](./testFiles-template/randomized/RandomizedTestCases.java) file that was copied to [testFiles](../../../src/main/resources/templates/java/test/testFiles) in Step 1.
```

Comment:
It would be nice to have a script for reverting the changes since if done manuallly it can break things?

Reply:
added scripts for both step 1 and 4 👍